### ISSUE TITLE
Accessibility Issue: <aside> must not be contained within another landmark /presentation.html

### ISSUE THREAD

**Issue by Moro-Code** [sara-sabr/ITStrategy#734]
**Description**
aside must not be contained within other landmarks 

**Device Information**  
*Device*:  
*Operating System (incl. version)*:  
*Browser (incl. version)*:  
*Additional Details*:

**Element(s) Information**  
*Page*:  https://sara-sabr.github.io/ITStrategy/presentation.html
*CSS Selector(s) (if possible and applicable)*:  aside 
*Additional Identifier(s) (if possible and applicable)*:  
*Additional Details*:  

**Screenshot(s) (if possible)**

**Additional information or thoughts**
As a result of scan #698 

https://dequeuniversity.com/rules/axe/3.4/landmark-complementary-is-top-level?application=AxeChrome

**Comment by Moro-Code** [comment]
Part of reveal.js itself and not WCAG 2.0 AA violation
